app/pages/ServiceDiscoveryResults/ServiceDiscoveryResults.tsx

```diff
@@ -1,21 +1,35 @@
-import React, { useState, useEffect } from "react";
```

Comment:
**suggestion (non-blocking):** What if we renamed these components to something like SearchExperience and BrowseExperience? It's getting hard to keep track of !

Reply:
Yes we need to do that, I thought the same thing.